charts/influxdb2/templates/statefulset.yaml

```diff
@@ -48,7 +48,7 @@ spec:
           - name: data
             mountPath: /root/.influxdbv2
           resources:
-            {{ toYaml .Values.resources | indent 12 }}
+            {{ .Values.resources | toYaml | nindent 12 | trim }}
```

Comment:
`nindent` is a typo, right?

Reply:
The `nindent` function is similar to the indent function, but begins with a newline. The fix works with `indent` as well, I can change it to that if desired :)

Reply:
TIL 😃

Reply:
You can find the documentation for it here:
http://masterminds.github.io/sprig/strings.html

Should I change it to `nindent` or would `indent` be fine? :)

Reply:
It's fine..the chart is failing to validate because `home` is missing from `Chart.haml`

I'm on mobile until Monday, if you add it, I can get it merged 👍

Thanks for your contribution and for teaching me a new function!

Reply:
Thanks for being so responsive and helpful!

I pushed another commit with the missing `home` in the `Chart.yaml`. GitHub has some degraded services right now, seems like it's not reflected in this PR yet.